client/components/PubCollectionsListing/PubCollectionsListing.tsx

```diff
@@ -196,19 +226,32 @@ const PubCollectionsListing = (props: Props) => {
 		);
 	};
 
+	const emptyListPlaceholderText = scopeData.activePermissions.canManageCommunity
+		? 'Create a new Tag with this name'
```

Comment:
As far as I can tell this text won't ever come up, because the `createNewItemRenderer` is always going to render a placeholder that prevents the list from being empty. Instead, I think you want to modify `renderNewItem` so that it tells users that clicking this item will create a new Collection:

```ts
renderCollectionTitle({ title: `Create new tag: ${query}`, ... })
```

Reply:
ngl, i stared at this piece of code for seven minutes wondering if i was going crazy because a formatted string was being passed with title...Then i tried it...then it worked....thank you?